force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -12,7 +12,7 @@ public class LeadTriggerHandler extends TriggerHandler {
         this.oldLeadMap = (Map<Id, Lead>) Trigger.oldMap;
     }
 
-    public override void beforeUpdate() {
+    public override void beforeUpdate,beforeInsert;{
```

Comment:
This can't work. Give it to back and declare new method: public override void beforeInsert() {         }

Reply:
ok